src/notebooks/telemetry/interpreterPackageTracker.ts

```diff
@@ -28,11 +28,18 @@ export class InterpreterPackageTracker implements IExtensionSyncActivationServic
         @inject(IDisposableRegistry) private readonly disposables: IDisposableRegistry,
         @inject(IInterpreterService) private readonly interpreterService: IInterpreterService,
         @inject(IPythonApiProvider) private readonly apiProvider: IPythonApiProvider,
-        @inject(IControllerRegistration) private readonly notebookControllerManager: IControllerRegistration
+        @inject(IControllerRegistration) private readonly notebookControllerManager: IControllerRegistration,
+        @inject(IsWebExtension) private readonly webExtension: boolean
     ) {}
     public activate() {
         this.notebookControllerManager.onControllerSelected(this.onNotebookControllerSelected, this, this.disposables);
-        this.interpreterService.onDidChangeInterpreter(this.trackPackagesOfActiveInterpreter, this, this.disposables);
+        if (!this.webExtension) {
```

Comment:
To be fixed properly here https://github.com/microsoft/vscode-jupyter/issues/13610